src/tactic/tauto.lean

```diff
@@ -28,6 +28,7 @@ do hs ← local_context,
          | `(¬ (_ ∧ _))  := replace h.local_pp_name ``(decidable.not_and_distrib'.mp %%h) <|>
                             replace h.local_pp_name ``(decidable.not_and_distrib.mp %%h)
          | `(¬ (_ ∨ _))  := replace h.local_pp_name ``(not_or_distrib.mp %%h)
+         | `(¬ _ ≠ _)      := replace h.local_pp_name ``(decidable.of_not_not %%h)
```

Comment:
Are these lines supposed to line up?
```suggestion
         | `(¬ _ ≠ _)    := replace h.local_pp_name ``(decidable.of_not_not %%h)
```

Reply:
Thanks Anne!  Given that CI has been building for almost two hours, I will wait to push your suggestion, so that I can see if there is the need to fix something else more quickly.

Note that all alignment will not be perfect, since some lines overflow in weird ways, but I "fixed" some more lines that could be synched as well.

Once this round of CI finishes, I will push this cosmetic change and will wait again for CI to build before merging.